packages/office-addin-cli/src/main.ts

```diff
@@ -2,5 +2,5 @@
 // Licensed under the MIT license.
 
 export * from "./log";
+export * from "./npmPackage";
```

Comment:
I renamed to "npmPackage.ts" since it is a more concise and description name for the category of functionality.